samples/net/sockets/http_server/src/ws.c

```diff
@@ -324,7 +324,7 @@ int ws_echo_setup(int ws_socket, struct http_request_ctx *request_ctx, void *use
 #define MAX_NAME_LEN sizeof("ws[xx]")
 		char name[MAX_NAME_LEN];
 
-		snprintk(name, sizeof(name), "ws[%d]", slot);
+		snprintk(name, sizeof(name), "ws[%d]", slot % (10 * MAX_NAME_LEN));
```

Comment:
This does not look right. The idea is to print the slot value here. If the name variable is too small, let's increase the size of it instead, so something like 
`MAX_NAME_LEN sizeof("ws[xxxx]")`

Reply:
I pushed the same idea as you said but did `sizeof("ws[]") + 10`

Reply:
Having +10 certainly works, but it is now a magical constant and not immediately obvious why it is there. Therefore we used the "template" way in other part of the code and I suggest we do that here too.